pages/common/git-fame.md

```diff
@@ -0,0 +1,28 @@
+# git fame
+
+> Pretty-print Git repository contributions.
+> More information: <https://github.com/casperdcl/git-fame>.
+
+- Calculate contributions for the current Git repository:
+
+`git fame`
+
+- Exclude files/directories that match the specified regular expression:
+
+`git fame --excl "{{regular_expression}}"`
+
+- Calculate contributions made after the specified date:
+
+`git fame --since "{{3 weeks ago|2021-05-01}}"`
```

Comment:
```suggestion
`git fame --since "{{3 weeks ago|2021-05-15}}"`
```
This would make clear which is the day and which the month for people who aren't used to that time format.

Reply:
Very good idea.

Reply:
I'll pick the 13th, because that's a very special day to me :)